### PR TITLE
wikipedia: modify command names

### PR DESCRIPTION
**Pull request by xnaas**

### CODE REVIEW

**Review by Exirel — APPROVED** [review]
:100: agree.

**Comment by dgw** [comment]
Since this is targeted for a major version bump… what if we were _really_ naughty and just nuked _all_ of the existing command names in favor of `.wikipedia` and `.wp`? Honestly, `.wiki` is a bit too generic—not as bad as `.w`, but still. 😜

I do have real reasoning behind this idea: Suppose someone wants to make a bot for some media franchise with a ton of very specific info available on a dedicated wiki (official or otherwise). Having `.wiki` available for use by plugins would let any fandom very easily make a bot that searches the de facto wiki for their niche, with an obvious command name that fits naturally.

**Comment by xnaas** [comment]
I know this came up in IRC, but I'll comment it here as well.

I think the general consensus is that `wp` sounds like wordpress and `wiki` is pretty much automatically assumed as wikipedia, but ¯\\\_(ツ)_/¯ Obviously go with whatever you think is best. It's your bot, not mine. 😏

**Review by dgw — CHANGES_REQUESTED** [review]
Just formalizing what I said in a comment months ago.

File: sopel/modules/wikipedia.py [hunk 1]
```diff
@@ -252,8 +252,8 @@ def mw_info(bot, trigger, match=None):
         say_snippet(bot, trigger, match.group(1), unquote(match.group(2)), show_url=False)
 
 
-@plugin.command('w', 'wiki', 'wik')
-@plugin.example('.w San Francisco')
+@plugin.command('wiki', 'wp')
+@plugin.example('.wiki San Francisco')
```
[the remark this answers is not in the document]
Comment (xnaas) [reply sopel/modules/wikipedia.py:256]: What about `wk` vs `wp`? 🤔 Similar to `wt`, I suppose.

[the remark this answers is not in the document]
Comment (dgw) [reply sopel/modules/wikipedia.py:256]: WaniKani? 😂

Jokes aside, `.wk` could be an abbreviation for just about _any_ Wikimedia project. They all have those two letters in the first three letters of the project name.

Wiktionary is a special case, really. Just about everything else follows the "Wikisomething" pattern, with both i's in the "wiki" part:

- Wikipedia
- Wikiquote
- Wikibooks
- Wikisource
- Wikispecies
- Wikinews
- Wikiversity
- Wikivoyage
- Wikidata

Ignoring the overlaps there (`ws` and `wv` could each represent two projects), the obvious abbreviation for each is `w_`, where the second character is what comes after the "Wiki" part.

I don't understand the resistance to using `wp`. 😕 The Wikipedia project itself uses that abbreviation as an internal shortcut—policy pages in the `Wikipedia:` namespace have short names like `WP:MOS`.

**Comment by RustyBower** [comment]
I agree. I think we should avoid single character commands as much as possible, there's too much opportunity for collision

File: sopel/modules/wikipedia.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (xnaas) [reply sopel/modules/wikipedia.py:256]: > I don't understand the resistance to using `wp`. 😕

Personally-motivated greed. 😜 — decade of habit/comfort. It's fine, though. Can always route my own `wk` command do it or something.

**Review by dgw — APPROVED** [review]
History cleaned. Didn't expect @xnaas of all people to be one of those Vigilant Mode nerds 😜 so feel free to `--amend` the commit here with a proper signature.

**Pull request merged**